src/commcare_cloud/commands/ansible/downtime.py

```diff
@@ -31,6 +31,11 @@ class Downtime(CommandBase):
         Argument('-m', '--message', help="""
             Optional message to set on Datadog.
         """),
+        Argument('-d', '--duration', default=24, help="""
```

Comment:
May be worth noting the default is 24 hours if not set and that the units are in hours